ledger/internal/eval.go

```diff
@@ -1041,13 +1005,18 @@ func (eval *BlockEvaluator) compactCertVotersAndTotal() (root crypto.Digest, tot
 	}
 
 	if voters != nil {
-		root = voters.Tree.Root()
-		total = voters.TotalWeight
+		root, total = voters.Tree.Root(), voters.TotalWeight
 	}
 
 	return
 }
 
+// TestingTxnCounter - the method returns the current evaluator transaction counter. The method is used for testing purposes
+// only.
+func (eval *BlockEvaluator) TestingTxnCounter() uint64 {
```

Comment:
we can emphasize this by passing `(*testing.TB)` as a param

Reply:
I'm not sure it's the best thing to do - I'd rather avoiding including the testing package in non-testing code.
And unfortunately, `_test` files cannot be access by `_test` files from different packages.